README.md

```diff
@@ -0,0 +1,56 @@
+# batwatch
+
+_batwatch_ monitors a system's batteries and UPS units for state changes and sends
+notifications via _gpgmailer_.
+
+Bug fixes are welcome!
+
+## System Power States
+
+_batwatch_ looks at the number of batteries installed on a system and each of those
+batteries' charge levels to determine the system's overall state to be one of the following.
+
+* Fully Charged: All batteries are fully charged and not discharging.
+* Charging: All batteries are either fully charged or charging.
+* Discharging: At least one battery is discharging.
+* No Battery: No batteries have been detected.
+
+A notification e-mail will be sent when the number of batteries installed changes or the
+overall charge state changes.
+
+## Prerequisites
+
+This software is currently only supported on Ubuntu 18.04 and may not be ready for use in a production environment.
+
+The only current method of installation for our software is building and installing your own debian package. We make the following assumptions:
+
+*    You are already familiar with using a Linux terminal.
+*    You are already somewhat familiar with using debuild.
+*    `debhelper` is installed.
+*    `devscripts` is installed.
```

Comment:
I'm unsure if we need to require both debhelper and devscripts.

Reply:
We do actually need debhelper for debuild to build the deb.